device-types/Raritan/PX3-5528V-V2.yaml

```diff
@@ -0,0 +1,94 @@
+---
+manufacturer: Raritan
+model: PX3-5528V-V2
+slug: px3-5528v-v2
+part_number: PX3-5528V-V2
+u_height: 0
+is_full_depth: false
```

Comment:
Please add the `weight` and `weight_unit` data point if you have/can find it

Reply:
weight and weight_unit added